style.css

```diff
@@ -0,0 +1,335 @@
+* { box-sizing: border-box; }
+
+body {
+  margin: 0 auto;
+  padding: 0;
+}
+
+main {
+  width: 375px;
+  margin: 12px auto;
+  align-items: flex-start;
+  flex-direction: column;
+  display: flex;
+  padding: 12px;
+}
+
+header {
+  width: 375px;
+  justify-content: space-between;
+  display: flex;
+  margin-bottom: 12px;
+  padding-right: 24px;
+}
+
+nav {
+  width: 15px;
+  height: 15px;
+  align-self: center;
+}
+
+.logo {
+  font-size: 18px;
+  align-items: center;
+  width: 76px;
+  height: 20px;
+  font-weight: bold;
+  font-family: 'Times New Roman', Times, serif;
+  font-style: normal;
+  line-height: 20px;
+  color: blue;
+  text-decoration: none;
+}
+
+.container {
+  padding-top: 70px;
+  margin-top: 12px;
+  width: 375px;
+  height: auto;
+  background-image: url(image/header-shapes\ mobile.png);
+}
+
+.one {
+  width: 327px;
+  height: 150px;
+  left: 24px;
+  margin-top: 114px;
+  display: flex;
+  font-size: 40px;
+  font-weight: bold;
+  font-style: normal;
+  font-family: serif;
+  line-height: 52px;
+  color: black;
+  align-items: center;
+}
+
+.two {
+  width: 327px;
+  left: 24px;
+  font-size: 16px;
+  font-style: normal;
+  font-family: Arial, Helvetica, sans-serif;
+  color: black;
+  line-height: 30px;
+  margin: 0 0;
+}
+
+.detail {
+  margin: 12px 0;
+}
+
+.two a {
+  font-family: Arial, Helvetica, sans-serif;
+  line-height: 24px;
+  font-size: 15px;
+  text-decoration: none;
+  font-weight: 400;
+  color: blue;
+  margin: 10px 0;
+}
+
+.nest {
+  width: 150px;
+  justify-content: space-between;
+  display: flex;
+  padding-inline-start: 0%;
+  list-style: none;
+  align-items: baseline;
+}
+
+.two p {
+  margin: 15px 0;
+}
+
+.cards {
+  margin: 25px auto;
+  display: grid;
+  grid-template-columns: auto;
+  grid-auto-flow: row;
+  width: 100%;
+  gap: 40px;
+  justify-content: center;
+}
+
+.card {
+  padding: 16px;
+  background: white;
+  border: 1px solid #dfe1e6;
+  border-radius: 15px;
+}
```

Comment:
- [ ] Great job so far 👍. However, there are some improvements you have to make. Although the layout of the card looks great, it's one of the project requirements to use `Flexbox` to layout individual cards, kindly fix that.
- [ ] There is a little style **(background color, font)** difference between your cards and the one on Figma.
- [ ] Please implement the button interaction like **hover, pressed**

Your Implementation | Figma Design
:---: | :---:
![image](https://user-images.githubusercontent.com/59416136/154497804-075ba525-46d6-4e42-9515-1158e3a2469f.png) | ![image](https://user-images.githubusercontent.com/59416136/154497939-30d6b837-6599-4ed7-82ac-2bf9cfa0d590.png)